.github/workflows/apply-template.yml

```diff
@@ -0,0 +1,68 @@
+name: Apply Template
+
+on:
+  push:
+    branches: ["main"]
+
+jobs:
+  repo_ids:
+    name: Get Repository Name
+    runs-on: ubuntu-latest
+    outputs:
+      is_renamed: ${{ steps.check_rename_occurred.outputs.IS_RENAMED }}
+      repo_name: ${{ steps.check_rename_occurred.outputs.REPO_NAME }}
+      org_name: ${{ steps.check_rename_occurred.outputs.ORG_NAME }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Check if rename has occurred
+        id: check_rename_occurred
+        run: |
+          LINES=$(grep openapi-ts-template ./package.json)
+          if [ -z "$LINES" ]; then
+            echo "IS_RENAMED" >> true"
+          else
+            echo "IS_RENAMED" >> false"
+          fi
+          REPO_NAME=$(echo "${{ github.event.repository.name }}" | tr '[:upper:]' '[:lower:]')
+          ORG_NAME=$(echo "${{ github.event.repository.owner.name }}" | tr '[:upper:]' '[:lower:]')
+          echo "REPO_NAME >> $REPO_NAME"
+          echo "ORG_NAME >> $ORG_NAME"
```

Comment:
`"REPO_NAME=$REPO_NAME" >> $GITHUB_OUTPUT` format?

Reply:
This is still not fixed.